docker/production.yml

```diff
@@ -30,6 +30,7 @@ services:
         - AUTH_URL=${AUTH_URL}
         - POSTS_URL=${POSTS_URL}
         - SEARCH_URL=${SEARCH_URL}
+        - FEED_DISCOVERY_URL=${FEED_DISCOVERY_URL}
```

Comment:
@humphd 
Do we need `'USERS_URL'` here?

Reply:
We should probably add it for future use.  But no, it's not needed now.

Reply:
Ok. 
I won't add the `USER_URL` for now.